### ISSUE TITLE
Fix dependency list

### ISSUE THREAD

**Issue by msolnit** [SOASTA/repository.js#23]
Here's my `npm install` output:

```shell
$ sudo npm install -g soasta-repository

npm WARN peerDependencies The peer dependency grunt@^0.4.0 included from grunt-jsdoc will no
npm WARN peerDependencies longer be automatically installed to fulfill the peerDependency 
npm WARN peerDependencies in npm 3+. Your application will need to depend on it explicitly.

/usr/local/bin/soasta-repository -> /usr/local/lib/node_modules/soasta-repository/cmd.js
soasta-repository@0.0.1 /usr/local/lib/node_modules/soasta-repository
├── commander@2.8.1 (graceful-readlink@1.0.1)
├── lodash@3.10.1
├── winston@1.0.2 (cycle@1.0.3, stack-trace@0.0.9, eyes@0.1.8, isstream@0.1.2, pkginfo@0.3.0, async@1.0.0, colors@1.0.3)
├── nock@2.13.0 (propagate@0.3.1, deep-equal@1.0.1, lodash@2.4.1, debug@1.0.4, mkdirp@0.5.1, chai@3.3.0)
├── grunt@0.4.5 (which@1.0.9, dateformat@1.0.2-1.2.3, eventemitter2@0.4.14, getobject@0.1.0, colors@0.6.2, rimraf@2.2.8, async@0.1.22, hooker@0.2.3, grunt-legacy-util@0.2.0, exit@0.1.2, nopt@1.0.10, lodash@0.9.2, coffee-script@1.3.3, underscore.string@2.2.1, iconv-lite@0.2.11, minimatch@0.2.14, glob@3.1.21, findup-sync@0.1.3, js-yaml@2.0.5, grunt-legacy-log@0.1.2)
└── grunt-jsdoc@0.6.10 (jsdoc@3.3.3, ink-docstrap@0.5.2)
```

Per @andreas-marschke some of these should be in `devDependencies` instead.

**Comment by msolnit** [comment]
@andreas-marschke are you sure about this?  Here's what `npm help install` has to say:

> By default, npm install will install all modules listed as dependencies.
> With the --production flag (or when the NODE_ENV environment variable
> is set to production), npm will not install modules listed in
> devDependencies.

**Comment by andreas-marschke** [comment]
Under normal conditions this is correct and looks like this: 
```
$> cd <repository.js-root>
$> npm install
// devDependencies and Dependencies downloaded
```
However when installing soasta-repository as a dependency for a project:
```
$> cd <other-project>
$> npm install soasta-repository
// soasta-repository and it's dependencies and _not_ it's devDependencies
```

That's what it is supposed to do but currently it also downloads the build dependencies (test-framework and ancillaries not required for normal soasta-repository usage).

**Comment by andreas-marschke** [comment]
Fix in #24

**Comment by andreas-marschke** [comment]
As #24 was merged we can close this.
